modules/json/JsonNatives.cpp

```diff
@@ -697,7 +697,7 @@ static cell AMX_NATIVE_CALL amxx_json_object_set_value(AMX *amx, cell *params)
 		return 0;
 	}
 
-	if (!JsonMngr->IsValidHandle(params[3], Handle_Object))
+	if (!JsonMngr->IsValidHandle(params[3]))
```

Comment:
This change appears unrelated.

Reply:
Removed, it just fix error with amxx for my purposes)